src/main/java/org/snomed/snowstorm/core/data/domain/Axiom.java

```diff
@@ -119,12 +120,49 @@ public Axiom setRelationships(Set<Relationship> relationships) {
 		return this;
 	}
 
-	@JsonIgnore
```

Comment:
I am not comfortable exposing the axiom refset member in the REST API. We are currently hiding this, and the OWL expression contained within, in favour of viewing and editing concepts in terms of axioms with a set of generated "relationships".
Editing the OWL expression within the refset and sending it back to the API will not work as intended so I think it's better to hide the refset member. CC @CoderMChu

Reply:
Thanks for your input @kaicode. 

I needed this as the member was not being written to the store as part of the "ManuallyMergedConcept" string. As a result, the autoMerge code was falling over with a NPE because there was no ReferenceSetMember associated with the Axiom.

Reply:
It's not okay to expose this refset member in the REST API. If it is being exposed by the removal of this annotation I think we should look for an alternative approach. 
The refset member id is probably already persisted so you could load it from the branch when applying the merge?

Reply:
Thanks both, I'll look into a different way.

Reply:
@kaicode @CoderMChu I have implemented another way. I now find the member from elasticsearch rather than exposing the member in the rest api. You can see this in the `joinAxiomReferenceSetMembers` method in AutoMerger.